README.md

```diff
@@ -103,11 +103,32 @@ Fedora 29, CentOS, RHEL, and related distributions:
 openSUSE:
 ```sudo zypper install cri-o```
 
-On Ubuntu distributions, there is a dedicated PPA provided by
-[Project Atomic](https://www.projectatomic.io/):
+Debian (10 and newer including Raspbian) and Ubuntu (18.04 and newer): Packages are available via the
+[Kubic](https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable)
+project repositories:
 
 ```bash
-sudo apt-add-repository ppa:projectatomic/ppa
+# Debian Unstable/Sid
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_Unstable/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_Unstable/Release.key -O- | sudo apt-key add -
+
+# Debian Testing
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_Testing/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_Testing/Release.key -O- | sudo apt-key add -
+
+# Debian 10
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_10/Release.key -O- | sudo apt-key add -
+
+# Raspbian 10
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Raspbian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Raspbian_10/Release.key -O- | sudo apt-key add -
+
+# Ubuntu (18.04, 19.04 and 19.10)
+. /etc/os-release
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/x${NAME}_${VERSION_ID}/Release.key -O- | sudo apt-key add -
+
 sudo apt-get update -qq
 sudo apt-get install cri-o-[REQUIRED VERSION]
```

Comment:
@saschagrunert @mrunalp @TomSweeneyRedHat btw, just wanna confirm the `[REQUIRED VERSION]` part is ok here. My initial PR had 1.16 for release-1.16 branch. The packages present on the repo are `cri-o-1.16` and `cri-o-1.17`.